### ISSUE TITLE
Flakey test: TestAccActiveDirectoryDomain_update

### ISSUE THREAD

**Issue by melinath** [hashicorp/terraform-provider-google#8660]
### Affected Resource(s)

* google_active_directory_domain

TestAccActiveDirectoryDomain_update is flakey. Here are a [few](https://ci-oss.hashicorp.engineering/buildConfiguration/GoogleCloud_ProviderGoogleCloudGoogleProject/172158?buildTab=tests&expandedTest=7699655873097682585) [example](https://ci-oss.hashicorp.engineering/buildConfiguration/GoogleCloud_ProviderGoogleCloudGoogleProject/172387?buildTab=tests&expandedTest=7699655873097682585) [failures](https://ci-oss.hashicorp.engineering/buildConfiguration/GoogleCloud_ProviderGoogleCloudGoogleProject/175693?buildTab=tests&expandedTest=7699655873097682585). Although this usually seems to get fixed on retry, it would be great to clean it up.

Failure rate:
- 13% in Aug 2021
- 27% in Sept 2021 (8 times)
- 22% in Dec 2021

**Comment by melinath** [comment]
Possibly related to https://github.com/hashicorp/terraform-provider-google/issues/7159

**Comment by rileykarson** [comment]
Let's wait on 7159 closing to re-evaluate this one.

**Comment by melinath** [comment]
13% failure rate in August

**Comment by mitj04** [comment]
b/261373951

**Comment by melinath** [comment]
Duplicate of https://github.com/hashicorp/terraform-provider-google/issues/16261
